### PR TITLE
Minor miscellaneous fixes

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: tests/integration/yocto/meta-mender-python-client/recipes-mender-python-client/mender-python-client/files/mender-sub-updater [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Run the original mender updater as a sub-updater to the mender-python-client
 #
-# This script is run by the API client through the symlink to /var/lib/mender/install
+# This script is run by the API client through the symlink to /usr/share/mender/install
```
Comment (oleorhagen) [inline tests/integration/yocto/meta-mender-python-client/recipes-mender-python-client/mender-python-client/files/mender-sub-updater:5]: Thank you!

File: src/mender/config/config.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ def __init__(self, global_conf: dict, local_conf: dict):
         log.debug("Mender configuration values:")
         for k, v in vals.items():
             if k == "ServerURL":
+                v = v.rstrip("/")
                 log.debug(f"ServerURL: {v}")
                 self.ServerURL = v
```
Comment (oleorhagen) [inline src/mender/config/config.py:42]: Suggestion:
```
self.ServerURL = v.rstrip("/")
```

**Review by oleorhagen — APPROVED** [review]
In short:
1. Minor suggestion
2. Thanks!
3. Yes!

File: src/mender/config/config.py (hunk 2 above)
Comment (lluiscampos) [reply src/mender/config/config.py:42]: We should log the exact same string we are saving. That was the thinking.

Comment (oleorhagen) [reply src/mender/config/config.py:42]: Oh, of course! Apologies. Go ahead

**Pull request merged**